src/hotspot/share/gc/g1/g1FullGCCompactionPoint.hpp

```diff
@@ -43,6 +44,7 @@ class G1FullGCCompactionPoint : public CHeapObj<mtGC> {
   void initialize_values();
   void switch_region();
   HeapRegion* next_region();
+  Pair<uint, uint> find_contiguous_before(HeapRegion* hr, uint num_regions);
```

Comment:
I think we can skip returning a `Pair` here. The second value is only used to check if we found any range and instead we could use something like `UINT_MAX` or the max number of regions to mark that.